### PR TITLE
feat: add optional mailpit configuration

### PR DESCRIPTION
**Pull request by WebsiteDeveloper**

### CODE REVIEW

File: devcontainer-example/docker-compose.yml [hunk 1]
```diff
@@ -20,6 +20,23 @@ services:
   #   volumes:
   #     - postgresql-data:/var/lib/postgresql/data
 
+  # Enable Mailpit if you need to test outgoing mail services
+  # See https://mailpit.axllent.org/
+  #  mailpit:
+  #    image: axllent/mailpit
+  #    container_name: mailpit
+  #    restart: always
```
Comment (revant) [inline devcontainer-example/docker-compose.yml:28]: Don't set restart: always. Other services don't set it.

**Review by revant — CHANGES_REQUESTED** [review]

**Comment by revant** [comment]
Greenmail allows SMTP as well as IMAP locally:

```yaml
  greenmail:
    image: greenmail/standalone:2.0.0
    ports:
      - 3025:3025
      - 3110:3110
      - 3143:3143
      - 3465:3465
      - 3993:3993
      - 8181:8080
```

**Comment by revant** [comment]
Either allow edit or remove the restart and container_name.

**Comment by WebsiteDeveloper** [comment]
@revant sorry for the delayed response i removed the container name and restart.

Concerning greenmail, the main reason why we use mailpit is because it has a webinterface which greenmail does not to my knowledge

**Review by revant — APPROVED** [review]

**Comment by revant** [comment]
> Concerning greenmail, the main reason why we use mailpit is because it has a webinterface which greenmail does not to my knowledge

It gives http rest api. You'll need to configure thunderbird or some mail client with `localhost:3025` for SMTP and `localhost:3143` for IMAP

**Pull request merged**